src/pages/JobVacancies/JobVacancies.tsx

```diff
@@ -1,181 +1,190 @@
 import React from "react";
 import Modal from "styled-react-modal";
 import { motion } from "framer-motion";
-import { jobData } from "utils/DummyVacancyData";
 import { pageTransitions, staggerTransitions, tableTransitions, modalTransitions } from "utils/Animations";
 import { VacancyCard, VacancyForm, VacancyFull } from "components";
 import "./JobVacancies.css";
 import "components/VacancyCard/VacancyCard.css";
 import { AppContext } from "context";
+import { IJobData } from "../../utils/DummyVacancyData";
 import { backgroundColor } from "themes/theme";
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faSearch } from '@fortawesome/pro-duotone-svg-icons';
 
 const StyledModal = Modal.styled`
-	width: 90vw;
-	height: 90vh;
-	top: '50%',
-	left: '50%',
-	right: 'auto',
-	bottom: 'auto',
-	marginRight: '-50%',
-	transform: 'translate(-50%, -50%)',
-	display: flex;
-	align-items: center;
-	justify-content: center;
-	background-color: ${backgroundColor};
-	overflow-y: scroll;
+    width: 90vw;
+    height: 90vh;
+    top: '50%',
+    left: '50%',
+    right: 'auto',
+    bottom: 'auto',
+    marginRight: '-50%',
+    transform: 'translate(-50%, -50%)',
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: ${backgroundColor};
+    overflow-y: scroll;
 `;
 
 export const JobVacancies: React.FC<{}> = (): React.ReactElement => {
-	const { openModal, setOpenModal, showVacancy } = React.useContext(AppContext);
-	const [search, setSearch] = React.useState<string>('');
-	const [searchFilter, setSearchFilter] = React.useState<string>('title');
+    const { openModal, setOpenModal, showVacancy } = React.useContext(AppContext);
+    const [search, setSearch] = React.useState<string>('');
+    const [searchFilter, setSearchFilter] = React.useState<string>('title');
+    const [vacancies, setVacancies] = React.useState<IJobData[]>([]);
 
-	const toggleModal = () => {
-		setOpenModal(!openModal)
-	}
+    React.useEffect(() => {
+        fetch('http://interview-server-heroku.herokuapp.com/vacancies', {headers: {'Content-Type': 'application/json'}})
+            .then((response) => response.json())
```

Comment:
I think we could combine these promises here.

.then((response) => {
    response.json()
    setVacancies
}